app/src/main/java/ir/kindnesswall/utils/extentions/String.kt

```diff
@@ -49,15 +49,7 @@ fun String.isValidVerificationCode(): Boolean {
 /**
  * A method for validate a string is only number or not
  */
-fun String.isNumeric(): Boolean {
-    var i = length
-    while (--i >= 0) {
-        if (!Character.isDigit(this[i])) {
-            return false
-        }
-    }
-    return true
-}
+fun String.isNumeric(): Boolean = this.toCharArray().all(Character::isDigit)
```

Comment:
You can use `CharSequence.all { it.isDigit() }`

Reply:
Done